packages/components/src/icon-button/index.js

```diff
@@ -25,6 +25,7 @@ function IconButton( props, ref ) {
 		tooltip,
 		shortcut,
 		labelPosition,
+		size,
```

Comment:
Another way to avoid this prop is to directly pass the `Icon` component as an icon prop. That said, I'm not sure what's best but I know changes to this component can be impactful, so we need to check with some plugins..

Reply:
Right, I will investigate all that. The challenge is that we need to support different sizes:
- `IconButton` for the patterns to pick is 48x48